composer.json

```diff
@@ -25,10 +25,12 @@
         "components/font-awesome": "^5.14.0",
         "dflydev/fig-cookies": "^3.0.0",
         "doctrine/dbal": "^2.7",
+        "dragonmantank/cron-expression": "^3.1.0",
         "franzl/whoops-middleware": "^2.0.0",
         "illuminate/bus": "^8.0",
         "illuminate/cache": "^8.0",
         "illuminate/config": "^8.0",
+        "illuminate/console": "^8.0",
```

Comment:
so the helpers were added back because illuminate/console makes use of them right ? a bit annoying after having created the `flarum/laravel-helpers`, I was under the impression that laravel's internals didn't use the helpers.

Reply:
Correct, that was an oversight: we missed that they are needed for fof console, and that we were planning to integrate fof console back. And given how much time we spent discussing, it is annoying that we have to revert that decision so quickly. Since they **are** needed though. we might as well simplify and get back of the extra package though, I suppose: I'm not really seeing an alternative (IMO having the `schedule:list` command is critical)